### DISCUSSION TITLE
Member-ordering of decorated properties for Angular

### DISCUSSION THREAD

**Discussion by dionatan-g** [typescript-eslint/typescript-eslint#7099] [Technical Discussions]
Hello,

Is there a way to set an order for specific decorated properties in Angular?
I would like to achieve the following order lint:

@output
@input (setter and getter together, one after the other)
@contentChildren
@contentChild
@viewChildren
@viewChild
@hostBinding
@hostListener

public fields
private fields

constructor
lifecycles (onInit, destroy etc)
public methods
public standalone getters
private methods

**Comment by bradzacher** [comment]
I saw you posted on discord.

Clarifying for future reference - [as per our contributing guide, please don't use discussions for support](https://typescript-eslint.io/contributing/discussions)
